pydantic_aioredis/model.py

```diff
@@ -30,7 +30,42 @@ class Model(_AbstractModel):
     thismodel:key
     """
 
-    _auto_sync = True
+    # if _auto_sync is set to true, the model will automatically sync to redis when a field is changed
+    _auto_sync = False
+    # if _auto_save is set to true, the model will automatically save to redis when it is created
+    _auto_save = False
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        if self._auto_save:
+            self.__save_from_sync()
+
+    def __setattr__(self, name: str, value: Any):
+        """
+        Overrides the default __setattr__ to allow for auto_sync
+        """
+        super().__setattr__(name, value)
+        if self._auto_sync and getattr(self, "_store", None) is not None:
+            self.__save_from_sync()
+
+    def __save_from_sync(self):
+        """
+        Calls the async save function from a sync context
```

Comment:
A way to call an `async` function from a `sync` context is to use the `syncer` package
```
from syncer import sync

async def bar():
  ...

def foo():
  sync(bar())
```

Reply:
Ty, I'll check it out